_pages/business-units/18f/projects-partners/how-we-collaborate.md

```diff
@@ -2,80 +2,74 @@
-* **Allow space for exploratory conversations.** We often work on hard issues without easy or immediate answers. Focusing on solutions too early can lead teams to [avoid systemic issues](https://nonprofitaf.com/2019/04/solutions-privilege-how-privilege-shapes-the-expectations-of-solutions-and-why-its-bad-for-our-work-addressing-systemic-injustice/) or shortcut necessary collaborative thinking. Scheduling regular, timeboxed space for revisiting [wicked problems](https://en.wikipedia.org/wiki/Wicked_problem) can help those larger questions fit in with other work productively.
+**Including agency partners in sprint rituals** is a powerful way to model how we work on a daily basis. Most teams include partners in most (or all) sprint rituals, with the ambition that the partners will lead and consider maintaining the meetings by the end of the engagement.
 
+**Further reading**
+* [Introduction to agile](https://docs.google.com/presentation/d/1492vzbeA2xXsajOYEugn3CzfRIZ3rCkrqQZ8F3_m8GM/edit#):lock:
+* [Overview of sprint ceremonies](https://docs.google.com/presentation/d/1OyU8-PL4-iQ9P_K21OWuoYYgLdWTDSSOuGvehIzpfVk/edit#):lock:
+
+## Are there times we shouldn’t collaborate with our partners?
+Though we prefer to collaborate with our partners whenever possible, there are some situations where we might limit access to documents and artifacts, or participation in conversations, to our internal 18F project team. For example:
+* **Design research** is a special case. Study data (data that’s directly collected from researchers about research participants and their context of use) should be shared strictly on a need-to-know basis in accordance with our [participant agreements](https://methods.18f.gov/participant-agreement/) and our [privacy impact assessment for design research](https://www.gsa.gov/cdnstatic/20181022%20-%20Design%20Research%20PIA_posted%20version.pdf). Be especially careful when sharing recordings/notes collected during stakeholder interviews.
+* **Working drafts, in some cases.** For example, if the partner is likely to be overwhelmed by or misunderstand an early/messy draft of a report or recommendation. We default to open, but use your best judgement about whether sharing early drafts of work or products will be more harmful than helpful. It’s recommended that you develop an agreed-upon convention for quickly knowing which files and folders to share with the partner.
+* **Notes from and candid conversations about our partner and about our internal team processes.** It’s important for 18F project teams to get together and discuss our project-level goals.  This can include candid reflections about the partner or our internal project team, which can help the team strategize how to achieve our goals. Overall, remember that it’s okay to have conversations without the partner in order to better serve them.
+
+# Making remote collaboration successful
```

Comment:
Feels like this should be part of "making the most of remote meetings" above.

Reply:
Slightly tweaked the order and nested within that section.